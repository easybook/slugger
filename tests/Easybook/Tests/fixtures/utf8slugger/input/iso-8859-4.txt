 !"#$%&'()*+,-./
0123456789:;<=>?
@ABCDEFGHIJKLMNO
PQRSTUVWXYZ[\]^_
`abcdefghijklmno
pqrstuvwxyz{|}~
ĄĸŖ¤ĨĻ§¨ŠĒĢŦŽ¯
°ą˛ŗ´ĩļˇ¸šēģŧŊžŋ
ĀÁÂÃÄÅÆĮČÉĘËĖÍÎĪ
ĐŅŌĶÔÕÖ×ØŲÚÛÜŨŪß
āáâãäåæįčéęëėíîī
đņōķôõö÷øųúûüũū˙