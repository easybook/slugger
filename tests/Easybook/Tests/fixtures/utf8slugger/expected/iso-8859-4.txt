
0123456789
abcdefghijklmno
pqrstuvwxyz
abcdefghijklmno
pqrstuvwxyz
aqrilsegtz
arilsegtnzn
aaaaaaaeiceeeeiii
dnokoooouuuuuuss
aaaaaaaeiceeeeiii
dnokooo-ouuuuuu